docker/compose/vast.yaml

```diff
@@ -16,6 +16,8 @@ services:
     ports:
       - ${VAST_WEB_PORT:-5160}:5160
       - ${VAST_PORT:-5158}:5158
+    entrypoint:
+      - tenzir-ctl
```

Comment:
I think this is a good workaround for now. I'm impressed/suprised the CI goes through with just this change, we probably miss some tests in the dependent compose scripts. Fully adapting the docker scripts for the new `tenzir` /  `tenzir-node` paradigm is a fair amount of work and I don't think we should undertake it right now.

Reply:
Yes, I'd rather keep it in place like this to keep it tested and avoid bit-rot.
The adaptation will be done in the coming weeks.